demo-ui/app/uk/gov/ons/addressIndex/demoui/views/multiMatch.scala.html

```diff
@@ -137,6 +137,12 @@ <h3 class="panel-title">@messages("multi.sfatext")</h3>
             </div>
             <div id="search-loading"></div>
             @results.map { bulkResp =>
+                    <div>
+                        <b>Number of addresses searched:</b> @{MatchTypeHelper.countSearched(bulkResp.bulkAddresses)} <br>
+                        <b>Single matches:</b> @{MatchTypeHelper.countSingles(bulkResp.bulkAddresses)}<br>
+                        <b>Multiple matches:</b> @{MatchTypeHelper.countMultiples(bulkResp.bulkAddresses)}<br>
+                        <b>No match:</b> @{MatchTypeHelper.countEmpty(bulkResp.bulkAddresses)}
+                    </div>
```

Comment:
The b tag is deprecated in HTML5 (though currently all browsers support it)

```html
Note: According to the HTML 5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the <h1> to <h6> tags, emphasized text should be denoted with the <em> tag, important text should be denoted with the <strong> tag, and marked/highlighted text should use the <mark> tag.

Tip: You can also use the CSS "font-weight" property to set bold text.
```
Number of matches: should ideally be messages("multi.numberofmatches")